autogen_gpt_diary.py

```
@@ -0,0 +1,124 @@
+from autogen_agentchat.agents import AssistantAgent
+from autogen_agentchat.conditions import MaxMessageTermination, TextMentionTermination
+from autogen_agentchat.teams import RoundRobinGroupChat
+from autogen_ext.models.openai import AzureOpenAIChatCompletionClient
+import os
+from dotenv import load_dotenv
+# pip install git+https://github.com/microsoft/autogen.git openai azure-identity
+
+load_dotenv()
+
+async def writer_workflow(task_prompt: str) -> str:
+    api_key = os.getenv('autogen_api_key')
```

Ruff (0.11.9)

Use capitalized environment variable `AUTOGEN_API_KEY` instead of `autogen_api_key`

(SIM112)